python/paddle/fluid/contrib/slim/quantization/imperative/qat.py

```diff
@@ -59,7 +60,8 @@ def __init__(self,
                  weight_quantize_type='abs_max',
                  activation_quantize_type='moving_average_abs_max',
                  moving_rate=0.9,
-                 quantizable_layer_type=['Conv2D', 'Linear']):
+                 quantizable_layer_type=['Conv2D', 'Linear'],
+                 is_calc_out_scale=False):
```

Comment:
For deploying the quantized model on CPU and GPU, QAT has to calculate out threshold for all quantized ops, so please delete input params `is_calc_out_scale `.

Reply:
Done.